bills/urls.py

```diff
@@ -0,0 +1,43 @@
+# pylint: disable=invalid-name
+"""Urls for bills application"""
+
+from django.urls import include, path
+from rest_framework.routers import DefaultRouter
+from rest_framework_nested.routers import NestedSimpleRouter
+
+from bills import models, serializers
+from bills.views import create_viewset
+
+
+router = DefaultRouter()
+router.register(
+    "participants",
+    create_viewset(serializers.ParticipantSerializer, models.Participant, False),
+)
+
+router.register(
+    "events", create_viewset(serializers.EventSerializer, models.Event, False)
+)
+event_router = NestedSimpleRouter(router, "events", lookup="event")
+event_router.register(
+    "participants",
+    create_viewset(serializers.ParticipantSerializer, models.Participant, True),
+    base_name="event-participants",
+)
+event_router.register(
+    "bills",
+    create_viewset(serializers.BillSerializer, models.Bill, True),
+    base_name="event-bills",
+)
+event_router.register(
+    "payments",
+    create_viewset(serializers.PaymentSerializer, models.Payment, True),
+    base_name="event-payments",
+)
+
+router.register("bills", create_viewset(serializers.BillSerializer, models.Bill, False))
```

Comment:
There are duplicated endpoints. Please remove not-nested ones.
![obraz](https://user-images.githubusercontent.com/3868440/71125739-d270ef00-21e7-11ea-89c1-929963be4bfe.png)
![obraz](https://user-images.githubusercontent.com/3868440/71125800-e9afdc80-21e7-11ea-83fc-3f74305c8579.png)

Reply:
Detail views have to stay. Otherwise nested views don't work. Unless you know how to bypass this problem?

Reply:
Done